solarwinds_apm/exporter.py

```diff
@@ -163,11 +165,15 @@ def _add_info_instrumented_framework(self, span, evt) -> None:
             and "opentelemetry.instrumentation" in instr_scope_name
         ):
             framework = instr_scope_name.split(".")[2]
-            # Some OTel instrumentation libraries are named not exactly
-            # the same as the instrumented libraries!
+            # Some OTel instrumentation scope names are not exactly
+            # the same as their corresponding instrumented libraries
             # https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/README.md
             if framework == "aiohttp_client":
```

Comment:
...while the package used for scope is at [opentelemetry.instrumentation.aiohttp_client](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-aiohttp-client/pyproject.toml#L40)